### PR TITLE
Add support for python 3.6

### PR DESCRIPTION
**Pull request by timleslie**
This change adds support for building pystache for python 3.6.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -42,7 +42,8 @@ test:
 
 about:
     home: http://github.com/defunkt/pystache
-    license: MIT License
+    license: MIT
+    license_file: LICENSE
```
Comment (jakirkham) [inline recipe/meta.yaml:47]: This will need a build number bump to get the license file included.

**Comment by jakirkham** [comment]
Might be worth giving [`noarch: python`]( http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages ) a try. This will avoid having to rebuild for new Pythons.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,6 +15,7 @@ source:
 build:
     number: {{ build }}
     script: python setup.py install --single-version-externally-managed --record record.txt
+    noarch: python
     entry_points:
         - pystache=pystache.commands.render:main
         - pystache-test=pystache.commands.test:main
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please add spaces around each `=` for the `entry_points`. Should fix the build issue.

**Comment by timleslie** [comment]
I think this PR is now properly ready for review. Thanks for your patience with all the incremental changes, this is my first time working with conda smithy.

**Comment by jakirkham** [comment]
Also would get rid of `bld.bat` in the `recipe`. That's a really old legacy workaround from some ancient `conda-build` 1.x that no longer exists.

**Comment by timleslie** [comment]
Hi @jakirkham, would you be able to review/merge this PR?

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
LGTM. Let's see what @kwilcox thinks.

**Comment by kwilcox** [comment]
I could have done with a rebase for a cleaner commit history... Next time!

**Comment by timleslie** [comment]
Thanks for the review @kwilcox, will definitely aim for a clean rebase next time, sorry about that!

**Comment by timleslie** [comment]
@kwilcox  It looks like the CircleCI tests failed on master after the PR merge, thought they were fine on this branch. Will this prevent the package ending up on conda-forge? Do these tests need to be retriggered somehow?

**Comment by jakirkham** [comment]
The SSH key used to checkout the code on CircleCI expired. We have refreshed it and it built through. Please let us know if there are other issues.

**Pull request merged**